packages/backend/src/open_payments/quote/service.test.ts

```diff
@@ -97,7 +97,7 @@ describe('QuoteService', (): void => {
     const paymentPointer = await createPaymentPointer(deps, {
       assetId: sendAssetId
     })
-    paymentPointerId = paymentPointer.id
+    sendingPaymentPointer = paymentPointer
```

Comment:
```suggestion
    sendingPaymentPointer = await createPaymentPointer(deps, {
      assetId: sendAssetId
    })
```

And we can remove:
https://github.com/interledger/rafiki/blob/a0fce7d005678b26bff3d99353600bbf277a21e5/packages/backend/src/open_payments/quote/service.test.ts#L97-L99

Reply:
Yup, for sure. Updated.